README.md

```diff
@@ -82,10 +86,15 @@ examples:
       goss:
         path: "/path/to/file.yml"
 
+    - name: test goss file
+      goss:
+        path: "/path/to/file.yml"
+        goss_path: "/usr/local/bin/goss"
+
     - name: test goss files
       goss:
         path: "{{ item }}"
         format: json
         output_file : /my/output/file-{{ item }}
         with_items: "{{ goss_files }}"
-```
```

Comment:
This three backticks are closing the source code block. We should keep this line.